src/Config/Config.ts

```diff
@@ -184,6 +184,11 @@ export class BridgeConfigGitLab {
     }
 }
 
+export interface BridgeConfigFeeds {
```

Comment:
Bonus points for updating the Config/Defaults.ts (which isn't really defaults, more of a sample config) so that users know how to configure it. Also make sure to regen the sample config.

Reply:
Ah, I did that manually, didn't notice it's generated. Fixed in ef91798.

Reply:
We should get a contributing file that explains all the magic things we do.